nilearn/input_data/nifti_labels_masker.py

```diff
@@ -181,30 +191,7 @@ def _check_fitted(self):
                              'You must call fit() before calling transform().'
                              % self.__class__.__name__)
 
-    def transform(self, imgs, confounds=None):
-        """Extract signals from images.
-
-        Parameters
-        ==========
-        imgs: Niimg-like object
-            See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
-            Images to process. It must boil down to a 4D image with scans
-            number as last dimension.
-
-        confounds: array-like, optional
-            This parameter is passed to signal.clean. Please see the related
-            documentation for details.
-            shape: (number of scans, number of confounds)
-
-        Returns
-        =======
-        signals: 2D numpy.ndarray
-            Signal for each region.
-            shape: (number of scans, number of regions)
-
-        """
-        self._check_fitted()
-
+    def transform_single_imgs(self, imgs, confounds=None):
```

Comment:
Just wondering whether you should not copy and paste the docstring from BaseMasker.transform_single_imgs here.